reference/constraints/IsFalse.rst

```diff
@@ -102,7 +102,7 @@ method returns **false**:
             }
         }
 
-        public function isStateInvalid()
+        function isStateInvalid()
```

Comment:
I think the method needs to be moved inside the class instead.

Reply:
Fixed!